src/Uno.UWP/Storage/Streams/DataWriter.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using System.IO;
+
+namespace Windows.Storage.Streams
+{
+	public partial class DataWriter : IDataWriter, IDisposable
+	{
+		// only one will be not null - depends on constructor used
+		private MemoryStream _memStream = null;
+		private IOutputStream _outputStream = null;
+		// z: fileOutputStream = detachStream();
+		// OutputStreamOverStream.cs , bez konstruktora, jako io.stream
+
+		private static int _unicodeOrder = 0;
+
+		public UnicodeEncoding UnicodeEncoding { get; set; }
+
+		public ByteOrder ByteOrder { get; set; }
+
+		public uint UnstoredBufferLength
+		{
+			get => (uint)(_memStream.Length - _memStream.Position);
+		}
+
+		//public DataWriterStoreOperation StoreAsync()
+		//{

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5527039)

Reply:
this is memo for anyone who would be implementing StoreAsync, which exception is to be thrown